tox.ini

```diff
@@ -1,18 +1,19 @@
 [tox]
-envlist = py3-docs
+envlist = docs
 skipsdist = true
+requires =
+  tox-pip-sync
+  tox-pyenv
+  tox-run-command
```

Comment:
In order this adds:

 * `tox-pip-sync` - Our new project to update your tox envs if the dependencies change
 * `tox-pyenv` - To make the version be determined by the `.python-version` file
 * `tox-run-command` - Required by `hdev` to work, but also quite handy as it lets you do `tox -e docs --run-command <ANYTHING YOU LIKE>`